### PR TITLE
Add examples of elements belonging to the palpable content category

### LINKED ISSUE #34199: Palpable content on the Content category page is somewhat incomplete.
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories

### What specific section or headline is this issue about?

https://html.spec.whatwg.org/multipage/dom.html#palpable-content

### What information was incorrect, unhelpful, or incomplete?

The [palpable content](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#palpable_content) does not include a list of elements that belong to that category.

### What did you expect to see?

It would be nice to add some example elements that are considered to be [palpable](https://html.spec.whatwg.org/multipage/dom.html#palpable-content) such as `<a>`, `<address>`, `<article>`, `<audio>` etc.

### Do you have any supporting links, references, or citations?

https://html.spec.whatwg.org/multipage/dom.html#palpable-content

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/content_categories`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/content_categories/index.md
* Last commit: https://github.com/mdn/content/commit/542c81306d3c34c32aabbf57a7a5d2c3b97b2d4c
* Document last modified: 2024-06-05T06:48:56.000Z

</details>

### PR DESCRIPTION
**Pull request by low-perry**
### Description

Adds the full list of elements belonging to the palpable content category as presented on the [specification](https://html.spec.whatwg.org/multipage/dom.html#palpable-content).

### Motivation

To complete missing information.

### Additional details

https://html.spec.whatwg.org/multipage/dom.html#palpable-content

### Related issues and pull requests

Fixes #34199

### CODE REVIEW

File: files/en-us/web/html/content_categories/index.md [hunk 1]
```diff
@@ -239,6 +239,76 @@ Some elements belong to this category only under specific conditions:
 ### Palpable content
 
 Content is palpable when it's neither empty nor hidden; it is content that is rendered and is substantive. Elements whose model is flow content should have at least one node which is palpable.
+Elements that belong to this category include:
```
Comment (pepelsbey) [inline files/en-us/web/html/content_categories/index.md:242]:
```suggestion
Content is palpable when it's neither empty nor hidden; it is content that is rendered and is substantive. Elements whose model is flow content should have at least one node which is palpable. Elements that belong to this category include:
```

File: files/en-us/web/html/content_categories/index.md [hunk 2]
```diff
@@ -239,6 +239,76 @@ Some elements belong to this category only under specific conditions:
...
+- {{SVGElement("svg")}} from [SVG](/en-US/docs/Web/SVG)
+- {{HtmlElement("table")}}
+- {{HtmlElement("textarea")}}
+- {{HtmlElement("time")}}
+- {{HtmlElement("u")}}
+- {{HtmlElement("var")}}
+- {{HtmlElement("video")}}
+- [Autonomus custom elements](/en-US/docs/Web/API/Web_components/Using_custom_elements)
+- text that is not inter-element [whitespace](/en-US/docs/Glossary/Whitespace)
+  Some elements belong to this category only under specific conditions:
```
Comment (pepelsbey) [inline files/en-us/web/html/content_categories/index.md:305]:
```suggestion
- text that is not inter-element [whitespace](/en-US/docs/Glossary/Whitespace)

Some elements belong to this category only under specific conditions:
```

**Review by pepelsbey — COMMENTED** [review]
Some formatting suggestions to keep it consistent. Otherwise, looks good!

File: files/en-us/web/html/content_categories/index.md (hunk 1 above)
Comment (low-perry) [reply files/en-us/web/html/content_categories/index.md:242]: Thank you.

**Review by pepelsbey — APPROVED** [review]
Perfect :) Thank you!

**Pull request merged**